### PR TITLE
ci: fix warnings in benchmark workflows

### PR DESCRIPTION
**Pull request by Uzlopak**
This fixes the deprecation warnings when running the benchmark workflow

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -86,8 +90,21 @@ jobs:
...
-          labels: |
-            benchmark
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          route: DELETE /repos/{repo}/issues/{issue_number}/labels/{name}
+          repo: ${{ github.event.pull_request.head.repo.full_name }}
+          issue_number: ${{ github.event.pull_request.number }}
+          name: benchmark
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```
Comment (Uzlopak) [inline .github/workflows/benchmark.yml:110]:
```suggestion

```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -86,8 +90,20 @@ jobs:
...
         with:
-          labels: |
-            benchmark
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          route: DELETE /repos/{repo}/issues/{issue_number}/labels/{name}
+          repo: ${{ github.event.pull_request.head.repo.full_name }}
+          issue_number: ${{ github.event.pull_request.number }}
+          name: benchmark
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Uzlopak) [inline .github/workflows/benchmark.yml:109]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -86,8 +90,20 @@ jobs:
...
-      - uses: actions-ecosystem/action-remove-labels@v1
+  remove-label:
+    if: "always()"
+    needs: 
+      - benchmark
+      - output-benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove benchmark label
+        uses: octokit/request-action@v2.x
```
Comment (Eomm) [inline .github/workflows/benchmark.yml:101]: actions-ecosystem/action-remove-labels was not good?

**Review by mcollina — APPROVED** [review]
lgtm

File: .github/workflows/benchmark.yml (hunk 3 above)
Comment (Uzlopak) [reply .github/workflows/benchmark.yml:101]: it is using node12
https://github.com/actions-ecosystem/action-remove-labels/blob/d05162525702062b6bdef750ed8594fc024b3ed7/action.yml#L24

So in the future, github will run node12 actions in node16 runtimes. You usually get the deprecation notice.

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**